hack/mocks/mock-config-2.yml

```diff
@@ -0,0 +1,6 @@
+---
```

Comment:
Why this file? Seems identical to the previous one.

Reply:
I have removed this and updated the code for fetching the curriculumDir.